source/_docs/autostart/systemd.markdown

```diff
@@ -66,7 +66,7 @@ After=docker.service
 [Service]
 Restart=always
 RestartSec=3
-ExecStart=/usr/bin/docker run --name=home-assistant-%i -v /home/%i/.homeassistant/:/config -v /etc/localtime:/etc/localtime:ro --net=host homeassistant/home-assistant
+ExecStart=/usr/bin/docker run --name=home-assistant-%i -v /home/%i/.homeassistant/:/config -v /etc/localtime:/etc/localtime:ro --net=host homeassistant/home-assistant:stable
```

Comment:
We should remove them altogether, as the proper way to handle this is using Docker itself, not systemd.

Reply:
Valid point. I didn't know that was built into Docker. 
1. Shall I make an issue for that and still push these for now? This change helped me get my install working.
2. Shall I add a note that Docker has its own restart policy and this is preferred? Link to the Docker docs maybe?

Reply:
Should I say to include `--restart=always` in the `docker run` command? With this link: https://docs.docker.com/config/containers/start-containers-automatically/

Reply:
Yes that is it.